### ISSUE TITLE
Attention mask is not set

### ISSUE THREAD

**Issue by cod3r0k** [anhnh2002/XTTSv2-Finetuning-for-New-Languages#50]
Hi, How can I set attention mask? I have a warning as below:

```

  0%|                                                                                                                                                   | 0/1 [00:00<?, ?it/s]
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass you

```
